pkg/operator/configobservation/etcdendpoints/observe_etcd_endpoints.go

```diff
@@ -1,105 +1,37 @@
 package etcdendpoints
 
 import (
-	"fmt"
-	"net"
-	"reflect"
-	"sort"
-	"strings"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 
 	"github.com/openshift/library-go/pkg/operator/configobserver"
+	libgoetcd "github.com/openshift/library-go/pkg/operator/configobserver/etcd"
 	"github.com/openshift/library-go/pkg/operator/events"
-	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-
-	"github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/configobservation"
-)
-
-const (
-	etcdEndpointNamespace = "openshift-etcd"
-	etcdEndpointName      = "etcd-endpoints"
 )
 
 // ObserveStorageURLs observes the storage config URLs. If there is a problem observing the current storage config URLs,
 // then the previously observed storage config URLs will be re-used.
 func ObserveStorageURLs(genericListers configobserver.Listers, recorder events.Recorder, currentConfig map[string]interface{}) (map[string]interface{}, []error) {
-	listers := genericListers.(configobservation.Listers)
-	oldStorageConfigURLsPath := []string{"storageConfig", "urls"}
-	newStorageConfigURLsPath := []string{"apiServerArguments", "etcd-servers"}
 	var errs []error
 
 	previouslyObservedConfig := map[string]interface{}{}
-	oldCurrentEtcdURLs, oldFound, err := unstructured.NestedStringSlice(currentConfig, oldStorageConfigURLsPath...)
+	oldCurrentEtcdURLs, oldFound, err := unstructured.NestedStringSlice(currentConfig, libgoetcd.OldStorageConfigURLsPath...)
```

Comment:
The library-go code has knowledge of these paths, why do we need to have this stanza here?